frontend/www/js/omegaup/submissions/list.ts

```diff
@@ -38,6 +46,31 @@ OmegaUp.on('ready', () => {
               })
               .catch(ui.apiError);
           },
+          'fetch-more-data': (username: string) => {
+            if (this.loading || this.endOfResults) return;
+            this.loading = true;
+            this.searchedUsername = username ?? null;
+
+            api.Submission.list({
+              username: this.searchedUsername,
```

Comment:
Here we could directly obtain the username from the `payload`:
```suggestion
              username: payload.username,
```
For this, it would be necessary for you to add this data in the `getLatestUserSubmissionsForTypeScript` function right where the `payload` object is.

Additionally, you need to add this new field to the `SubmissionsListPayload` type declaration. The field to add should be something like: `username?: string`.